api/vm/legacy_compiler.ts

```diff
@@ -26,6 +26,7 @@ const tableContextPropertyNames = [
   "disabled",
   "redshift",
   "bigquery",
+  "sqldatawarehouse",
```

Comment:
i'd leave out touching any of the legacy stuff, we want to remove this and definitely no new users should be using it